solr/solr-ref-guide/modules/query-guide/pages/stream-tool.adoc

```diff
@@ -71,13 +72,15 @@ CORSAIR ValueSelect 1GB 184-Pin DDR SDRAM Unbuffered DDR 400 (PC 3200) System Me
 A-DATA V-Series 1GB 184-Pin DDR SDRAM Unbuffered DDR 400 (PC 3200) System Memory - OEM
 ----
 
-The `--help` (or simply `-h`) option will output information on its usage (i.e., `bin/solr stream --help)`.
-
 == Using the bin/solr stream Tool
 
+NOTE: The Stream Tool is classified as "experimental".
```

Comment:
Might be worth putting this at the top of the page.  Seems a little "buried" as-is.

Reply:
love it